src/plugins/dashboard/public/application/utils/use/use_dashboard_app_state.tsx

```diff
@@ -23,12 +32,19 @@ import { SavedObjectDashboard } from '../../../saved_dashboards';
 export const useDashboardAppAndGlobalState = (
```

Comment:
I think we should probably rename this function and the comment something like `use_state_management()`? since it is taking care of global state, app state and embeddable container altogether now